sccm/apps/deploy-use/import-export-applications.md

```diff
@@ -0,0 +1,46 @@
+---
+
+# Mandatory fields.
+title: Import/Export Applications
+titleSuffix: Configuration Manager
+description: Import and export applications.
+author: chriskibble
+ms.author: NO VALUE SET # Microsoft employees only
+ms.date: 5/6/2019
+ms.topic: conceptual
+ms.technology: configmgr-app
+---
+# Create applications in Configuration Manager
+
+*Applies to: System Center Configuration Manager (Current Branch)*
+
+Configuration Manager allows for the import and export of applications within the console.  This allows you to copy applications in to other environments, such as copying from a test environment into a production environment.
+
+## <a name="bkmk_export"></a> Export an Application
```

Comment:
Use sentence case, and the anchor isn't always necessary. Actually, since this article is in the context of apps, then I think the "an application" part is redundant. So this header can be really simplified to just:

```suggestion
## Export
```

Reply:
Fixed in #1585